tests/forte/data/data_store_test.py

```diff
@@ -142,42 +143,51 @@ def setUp(self) -> None:
                 },
                 "parent_class": set(),
             },
-        }
-
-        self.base_type_attributes = {
-            'forte.data.ontology.top.Generics': {'parent_class': {'Entry'}},
-            'forte.data.ontology.top.Annotation': {'parent_class': {'Entry'}},
-            'forte.data.ontology.top.Link': {'parent_class': {'BaseLink'}},
-            'forte.data.ontology.top.Group': {'parent_class': {'Entry', 'BaseGroup'}},
-            'forte.data.ontology.top.MultiPackGeneric': {'parent_class': {'Entry', 'MultiEntry'}},
-            'forte.data.ontology.top.MultiPackLink': {'parent_class': {'MultiEntry', 'BaseLink'}},
-            'forte.data.ontology.top.MultiPackGroup': {'parent_class': {'Entry', 'MultiEntry', 'BaseGroup'}},
-            'forte.data.ontology.top.Query': {'parent_class': {'Generics'}},
-            'forte.data.ontology.top.AudioAnnotation': {'parent_class': {'Entry'}}
-        }
-
-        DataStore._type_attributes["ft.onto.base_ontology.Document"] = {
+            "forte.data.ontology.top.BoundingBox": {
                 "attributes": {
-                    "document_class": 4,
-                    "sentiment": 5,
-                    "classifications": 6,
+                    "_cy": 4,
```

Comment:
One reason that we don't want these to be `protected` is also because of the serialization. We don't want the serialization format to have these extra `_`